src/core.js

```diff
@@ -478,13 +478,7 @@ function extractStacktrace( e, offset ) {
 
 	var stack, include, i;
 
-	if ( e.stacktrace ) {
-
-		// Opera 12.x
-		return e.stacktrace.split( "\n" )[ offset + 3 ];
-	} else if ( e.stack ) {
-
-		// Firefox, Chrome, Safari 6+, IE10+, PhantomJS and Node
```

Comment:
You wouldn't mark that with a support comment; the existing comment seems fine. However, it's not at all clear what happens in the unsupported browsers. That should be documented in a comment.